src/streamlink/plugins/afreeca.py

```diff
@@ -224,10 +228,17 @@ def _get_streams(self):
         self.author = channel.get("BJNICK")
         self.title = channel.get("TITLE")
 
+        streams: Dict[str, HLSStream] = {}
```

Comment:
The typing information is useless here, because the entire method is not type-checked. Type-checking plugin implementations like this is usually irrelevant.